Foundation/include/Poco/Task.h

```diff
@@ -98,6 +98,8 @@ class Foundation_API Task: public Runnable, public RefCountedObject
 		/// If task has been cancelled prior to this call, it only sets
 		/// the state to TASK_FINISHED and notifies the owner.
 
+	bool hasOwner() const;
+		/// Returns true iff the task has an owner.
```

Comment:
Typo: iff

Reply:
actually, it is [intentional](https://www.merriam-webster.com/dictionary/iff)

Reply:
One can learn something new every day. 😀